acceptance_tests/features/display_respondent_alongside_response_status.feature

```diff
@@ -0,0 +1,42 @@
+@standalone
+Feature: Internal user can search for a respondent via email
+  As an internal user
+  I need to be able to search for a registered respondent via their email
+  So that I can easily find them when dealing with operational calls
+
+
+  @fixture.setup.data.with.enrolled.respondent.user.and.internal.user.and.new.iac.and.collection.exercise.to.live
```

Comment:
If you have the same tag on all scenarios it can be moved to the feature at the top of the file, it is treated as being on all scenarios in the file.